src/error.rs

```diff
@@ -1,4 +1,7 @@
 use math;
+use std::f64;
+
+const SQRT_PI: f64 = 1.772453850905515881919427556567825376987457275391f64;
```

Comment:
Is the suffix needed in this case?

Reply:
Surprised that `std::f64::consts` doesn’t have it.

Reply:
The suffix is not needed, but that is the convention used in `std::f64::consts`. Happy to remove it.